dask/bag/tests/test_random.py

```diff
@@ -51,6 +51,20 @@ def test_choices_k_equal_bag_size_with_unbalanced_partitions():
     assert all(i in seq for i in li)
 
 
+def test_choices_with_more_bag_partitons():
+    # test with npartitions > split_every (split_every=8 by default)
```

Comment:
This function currently doesn't take a `split_every` parameter (though we could easily retrofit it to do that)

Reply:
`split_every` is a parameter in `Bag.reduction()`, which might be an implementation detail. So, should we consider adding it to user-facing `sample`/ `choice`?

Reply:
Ah, I see. Thank you both for clarifying. @pavithraes would you mind adding a `split_every` keyword then? We already expose that keyword in lots of other similar bag methods and it would also have the added benefit of making related tests more robust to changes in the default value for `split_every`